frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -104,9 +135,184 @@ describe("WMG tissue auto-expand", () => {
+   * Blood selected. Only Abdomen and Blood should be visible. Abdomen should
+   * remain collapsed. Remove Sex filter. Tissue filter should now only have
+   * Abdomen and Blood selected. Only Abdomen and Blood should be visible.
+   * Abdomen should remain collapsed.
+   */
+  test("Tissue auto expansion - cross filter with Sex filter, check expansion", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterTissues(page, FILTERED_TISSUES);
+    await collapseTissue(page, FILTERED_TISSUES[0]);
+    await filterSelection(
+      page,
+      SEX_FILTER_TEST_ID,
+      SEX_FILTER_LABEL,
+      SEX_FILTER_SELECTION
+    );
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER.length
+    );
+    await checkTissues(
+      page,
+      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER,
+      EXPECTED_EXPANDED_TISSUES
+    );
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter selected
+   * Filter 'Ahren et al. Cell 2022' from the Publication filter. Filter Blood
+   * from the tissue filter. Blood should be expanded. Collapse Blood. add Cell
+   * filter for B Cell. Only B Cell should appear and Blood should be expanded.
+   */
+  test("Tissue auto expansion - cross filter with Publication filter selected", async ({
+    page,
+  }) => {
+    const tissues = ["blood"];
+    await loadPageAndTissues(page);
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[0]
+    );
+    await filterTissues(page, tissues);
+    await collapseTissue(page, tissues[0]);
+    await filterCellType(page, 1);
+    await checkTissues(page, tissues);
+    await checkElementVisible(page, EXPECTED_VISIBLE_CELL, CELL_TYPE_TEST_ID);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter, check expansion
+   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Blood. Filter
+   * 'Ahren et al. Neuron 2021' from the Publication filter. Only Blood should remain
+   * visible and collapsed. Add Cell filter for B Cell. Only B Cell should be
+   * visible under expanded Blood tissue node
+   */
+  test("Tissue auto expansion - cross filter with Publication filter, check expansion", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterTissues(page);
+    await collapseTissue(page, "blood");
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[1]
+    );
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(1);
+    await checkTissues(page, ["blood"], []);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Self-Reported Ethnicity filter
+   * Expand Breast. Select African from Ethnicity filter. Breast should remain expanded.
+   * Add Nose to the Tissue filter. Nose should appear expanded.
```

Comment:
I can't tell if we're adding Nose tissue in the test, but so far my impression is it's not in the test steps 🤔

Reply:
yes corrected the code to match the comment